packages/playwright-test/src/reporters/html.ts

```diff
@@ -140,6 +147,20 @@ function reportFolderFromEnv(): string | undefined {
   return undefined;
 }
 
+function getHtmlReportOptionProcessEnv(): HtmlReportOpenOption | undefined {
+  const processKey = 'PW_TEST_HTML_REPORT_OPEN';
+  const htmlOpenEnv = process.env[processKey];
+  if (htmlOpenEnv){
+    if (isHtmlReportOption(htmlOpenEnv)){
+      return htmlOpenEnv;
+    } else {
+      console.log(colors.red(`Configuration Error: HTML reporter Invalid value for ${processKey}: ${htmlOpenEnv}. Valid values are: ${htmlReportOptions.join(', ')}`));
+      return undefined;
+    }
+  }
+  return undefined;
+}
+
```

Comment:
```suggestion
function getHtmlReportOptionProcessEnv(): HtmlReportOpenOption | undefined {
  const processKey = 'PW_TEST_HTML_REPORT_OPEN';
  const htmlOpenEnv = process.env[processKey];
  if (!htmlOpenEnv)
    return undefined;
  if (!isHtmlReportOption(htmlOpenEnv)) {
    console.log(colors.red(`Configuration Error: HTML reporter Invalid value for ${processKey}: ${htmlOpenEnv}. Valid values are: ${htmlReportOptions.join(', ')}`));
    return undefined
  }
  return htmlOpenEnv;
}

```


Negating it is usually more readable and less nested.

Reply:
Sounds good!